### ISSUE TITLE
CosFace: Large Margin Cosine Loss for Deep Face Recognition

### ISSUE THREAD

**Issue by haru-333** [haru-333/paper#4]
# INFO

## link
[arXiv]()
[実装]()

# 概要

- 通常のクラス分類問題を学習させるだけで距離学習が実現できる
- 学習が容易なクラス分類モデルに1層独自のレイヤを追加するだけで、通常のクラス分類問題として学習が可能

# 提案手法
 CosFace, ArcFaceの違い
これらの手法の違いはペナルティのかけ方にあります。
logitsの値に対してcos(m1θyi+m2)−m3cos⁡(m1θyi+m2)−m3のようなペナルティを定義すると、m1m1がSphereFace、m2m2がArcFace、m3m3がCosFaceに対応します。
CosFace, ArcFaceの違いは、どの空間で綺麗にマージンが定義されるかの違いです。

![スクリーンショット 2022-09-06 13 31 52](https://user-images.githubusercontent.com/81456240/188547407-265779a7-581d-43df-960b-f7c69b513066.png)

# 検証

# 新規性

# 議論，展望

# Comment

# date
